manager/volume.go

```diff
@@ -552,14 +552,15 @@ func (m *VolumeManager) Expand(volumeName string, size int64) (v *longhorn.Volum
 		return nil, err
 	}
 
-	logrus.Infof("Volume %v expansion from %v to %v requested", v.Name, v.Spec.Size, size)
 	v.Spec.Size = size
 
 	v, err = m.ds.UpdateVolume(v)
 	if err != nil {
 		return nil, err
 	}
 
+	logrus.Infof("Expanding volume %v from %v to %v requested", v.Name, v.Spec.Size, size)
```

Comment:
`v.Spec.Size` is the same as `size`. You should log with the original volume size instead.